### PR TITLE
Created dedicated md file for TKO SaaS contents for RA

### PR DESCRIPTION
**Pull request by manishjha86**
Created dedicated md file for TKO SaaS contents for RA

### CODE REVIEW

**Comment by craigtracey** [comment]
HI @manishjha86 please take a look at the CLA requirements above as that will be necessary before merge. Thanks.

File: src/reference-designs/tko-saas.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+## Tanzu Mission Control
+
+Tanzu Mission Control is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds. It provides operators with a single control point to give developers the independence they need to drive business forward, while enabling consistent management and operations across environments for increased security and governance.
+
+### Attaching Tanzu Kubernetes Clusters to Tanzu Mission Control
+
+VMware recommends that you attach the shared services and workload clusters into Tanzu Mission Control as it provides a centralized administrative interface that enables you to manage your global portfolio of Kubernetes clusters. If the Tanzu Kubernetes Grid clusters are behind a proxy, import the proxy configuration to Tanzu Mission Control and then attach the cluster using Tanzu Mission Control UI or CLI.
```
Comment (craigtracey) [inline src/reference-designs/tko-saas.md:11]:
```suggestion
VMware recommends that you attach the shared services and workload clusters into Tanzu Mission Control as it provides a centralized administrative interface that enables you to manage your global portfolio of Kubernetes clusters. If the Tanzu Kubernetes Grid clusters are behind a proxy, import your proxy configuration to Tanzu Mission Control and then attach the cluster using Tanzu Mission Control UI or CLI.
```

File: src/reference-designs/tko-saas.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+# Tanzu for Kubernetes Operation SaaS Integration
+
+The SaaS products in the VMware Tanzu portfolio are in the critical path for securing systems at the heart of your IT infrastructure. VMware Tanzu Mission Control provides a centralized control plane for Kubernetes, and Tanzu Service Mesh provides a global control plane for service mesh networks. VMware Tanzu Observability by Wavefront offers observability and analytics for multi-cloud Kubernetes environments.
```
Comment (craigtracey) [inline src/reference-designs/tko-saas.md:3]:
```suggestion
The SaaS products in the VMware Tanzu portfolio are critical components for securing your IT infrastructure. VMware Tanzu Mission Control provides a centralized control plane for Kubernetes, and Tanzu Service Mesh provides a global control plane for service mesh networks. VMware Tanzu Observability by Wavefront offers observability and analytics for multi-cloud Kubernetes environments.
```

File: src/reference-designs/tko-saas.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
+# Tanzu for Kubernetes Operation SaaS Integration
```
Comment (craigtracey) [inline src/reference-designs/tko-saas.md:1]:
```suggestion
# Tanzu for Kubernetes Operations SaaS Integration
```

File: src/reference-designs/tko-saas.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+## Tanzu Mission Control
+
+Tanzu Mission Control is a centralized management platform for consistently operating and securing your Kubernetes infrastructure and modern applications across multiple teams and clouds. It provides operators with a single control point to give developers the independence they need to drive business forward, while enabling consistent management and operations across environments for increased security and governance.
+
+### Attaching Tanzu Kubernetes Clusters to Tanzu Mission Control
+
+VMware recommends that you attach the shared services and workload clusters into Tanzu Mission Control as it provides a centralized administrative interface that enables you to manage your global portfolio of Kubernetes clusters. If the Tanzu Kubernetes Grid clusters are behind a proxy, import the proxy configuration to Tanzu Mission Control and then attach the cluster using Tanzu Mission Control UI or CLI.
+**Note:** Registering Tanzu Kubernetes Grid 1.4 management cluster to Tanzu Mission Control is not supported.
```
Comment (craigtracey) [inline src/reference-designs/tko-saas.md:12]:
```suggestion
**Note:** Registering Tanzu Kubernetes Grid 1.4 management clusters to Tanzu Mission Control is not supported.
```

**Review by craigtracey — CHANGES_REQUESTED** [review]
I have made a few suggestions that I believe should be addressed. 

Additionally, this change has broken tests that will need to be addressed. First the broken link check is failing on `toc.md` as the file has been renamed. The orphaned file check is failing because we now have image files that are not being referenced any longer. Those files will need to be deleted.

To verify that your changes have fixed testing, please run `act` locally as described in [TESTING](https://github.com/vmware-tanzu-labs/tanzu-validated-solutions/blob/main/TESTING.md#running-tests)

**Comment by craigtracey** [comment]
@srinivasme21 please take a look for IX. Specifically, for how this will affect the TOC,

**Review by MarilynHorn — CHANGES_REQUESTED** [review]
See my general commit below about redoing the following 3 graphics to make them easier to read:

TSM-APP-CONNECTIVITY.PNG
TSM-OBSERVABILITY.PNG
TSM-RESILIANCY.PNG

**Comment by craigtracey** [comment]
@manishjha86 @srinivasme21 bump

**Review by MarilynHorn — CHANGES_REQUESTED** [review]
The only md file that requires changes is 'tko-saas.md'. Most of my suggestions are to conform more closely to the VMware writing guidelines.

The following 3 png files should be redone so that they are easier to read:

'TSM-APP-CONNECTIVITY.PNG'
'TSM-OBSERVABILITY.PNG'
'TSM-RESILIANCY.PNG'

Here are suggestions for making the graphics easier to read:

1. If possible, change to a light theme and recapture. It's much easier to read dark text against a light background than light text against a dark background. (There has been a lot of research about this.)
1. Before capturing the screen, make the text larger.
1. Capture just the part of the screen that's relevant. You probably don't need the sidebar on the left, much of the text above the diagram, and the black space below the diagram. On Windows, use the Snipping Tool to capture a rectangle.  On a Mac, press **Shift**, **Command**, and **4** to capture a rectangular area. Alternatively, you can capture the full screen and use a graphics editor to crop it.

**Review by craigtracey — APPROVED** [review]

**Pull request merged**